scep/context.go

```diff
@@ -0,0 +1 @@
+package scep
```

Comment:
Empty file?

Reply:
Removed in ba72710e2d11295287a9816a6d3dbd2b544ff075.